test/TestUserAcceptance.hs

```diff
@@ -670,14 +676,24 @@ tearDown (Env testdir _ sessionName) = do
 setUp :: Int -> String -> IO Env
 setUp i desc = do
   let
-    sessionName = intercalate "-" (sessionNamePrefix : show i : descWords)
+    num = show i
+    sessionName = intercalate "-" (sessionNamePrefix : num : descWords)
     descWords = words $ filter (\c -> isAscii c && (isAlphaNum c || c == ' ')) desc
   setUpTmuxSession sessionName
   prepareEnvironment sessionName
   (testdir, maildir) <- setUpTempMaildir
   setUpPurebredConfig testdir
+  precompileConfig testdir
   pure $ Env testdir maildir sessionName
 
+precompileConfig :: FilePath -> IO ()
+precompileConfig testdir = do
+  env <- getEnvironment
+  let purebredConfig = unionBy (\x y -> fst x == fst y) [("PUREBRED_CONFIG_DIR", testdir)] env
+      config = setEnv purebredConfig $ proc "stack" ["exec", "purebred", "--", "--version"]
+  _ <- readProcessWithErrorOutput_ config
```

Comment:
Why do you use `readProcessWithErrorOutput_` when we throw the result away?

Reply:
Hm ... yeah. Will change.